package.json

```
@@ -98,7 +98,7 @@
     "@types/node": "8.5.2",
     "angular2-template-loader": "0.6.2",
     "babel-loader": "8.0.4",
-    "carbon-components": "9.68.16",
+    "carbon-components": "^9.70.0",
```

We err on the side of caution and pin all our direct/dev deps

```suggestion
    "carbon-components": "9.70.0",
```